examples/dem/3d-plate-discharge/pyvista_utilities.py

```diff
@@ -0,0 +1,236 @@
+import numpy as np
+import pandas as pd
+import scipy as sc
+
+def position_string_to_int(coord_str):
+    string_to_int = {'x': 0, 'y': 1, 'z': 2}
+    return string_to_int[coord_str]
+
+class PlateDischargeUtilities:
+    def __init__(self, dataframe, prm_dict, time_list,  plate_norm, plate_direction):
```

Comment:
I would maybe add a quick comment on the constructor just to explain what you are doing in the constructor.